scripts/ci/build.sh

```diff
@@ -5,7 +5,7 @@ ROOTDIR="$(dirname "$0")/../.."
 cd "${ROOTDIR}"
 
 # shellcheck disable=SC2166
-if [ "$CIRCLE_BRANCH" = release -o -n "$CIRCLE_TAG" -o -n "$FORCE_RELEASE" ]; then echo -n >prerelease.txt; else date -u +"nightly.%Y.%-m.%-d" >prerelease.txt; fi
```

Comment:
Where do we make sure that this is called "nightly" for the actual emscripten build?

Reply:
Those are build in solc-bin from this github action https://github.com/ethereum/solc-bin/blob/gh-pages/.github/workflows/nightly-emscripten.yml

Reply:
Hm... which just calls back ``scripts/travis-emscripten/build_emscripten.sh`` ultimately...

Reply:
Maybe we actually don't :-) - I'll have a look and make sure - in any case, they won't involve this script, so they're unaffected by the change here.

Reply:
Yeah, we don't - they are called ``develop``... but that needs to be fixed in solc-bin. I'll create a PR there.